bin/server.py

```diff
@@ -50,7 +50,9 @@
 defaults_file = os.environ.get("WEBSERV_CONFIG", "~/.lsst/webserv.ini")
 WERKZEUG_PREFIX = "dax.webserv.werkzeug."
 
-app = Flask(__name__)
+# instance folder not under version control
+i_path=os.path.join(os.path.expanduser("~"), ".lsst/instance")
+app = Flask(__name__, instance_path=i_path, instance_relative_config=True)
```

Comment:
As reminder, In flask parlance, 'instance' = 'private' or 'not shared'.  this will come in handy if instances of the imgserv are to be deployed, say, in a flask cluster with access to different data sources and other config info.  To be discussed...
.